examples/demo/src/products/GridList.js

```diff
@@ -106,7 +106,7 @@ const LoadedGridList = ({ ids, data, basePath, width }) => {
     );
 };
 
-const GridList = ({ loadedOnce, ...props }) =>
-    loadedOnce ? <LoadedGridList {...props} /> : <LoadingGridList {...props} />;
```

Comment:
Isn't it a breaking change too for custom views ?

Reply:
Not sure I understand your remark, but yes, it is a breaking change, hence the new section in the upgrade guide.

Reply:
The new section says nothing about `loadedOnce`, only `isLoaded`

Reply:
my bad, it's `loadedOnce`, not `isLoaded`. I'll fix the upgrade guide right away.